gormigrate.go

```diff
@@ -178,7 +194,7 @@ func (g *Gormigrate) createMigrationTableIfNotExists() error {
 		return nil
 	}
 
-	sql := fmt.Sprintf("CREATE TABLE %s (%s VARCHAR(255) PRIMARY KEY)", g.options.TableName, g.options.IDColumnName)
+	sql := fmt.Sprintf("CREATE TABLE %s (id INTEGER PRIMARY KEY AUTOINCREMENT, %s VARCHAR(255))", g.options.TableName, g.options.IDColumnName)
```

Comment:
It can be switched to the native gorm AutoMigration flow and dialects will take care about this stuff. Also it will take care of new autoincrement column without any help from the user.